packages/orbit-components/src/Separator/index.tsx

```diff
@@ -79,7 +79,7 @@ const Separator = ({
         )}
       />
       {label && (
-        <span className="bg-white-normal px-100 absolute left-1/2 top-1/2 min-w-max -translate-x-1/2 -translate-y-1/2">
+        <span className="bg-white-normal px-100 absolute left-1/2 top-1/2 flex min-w-max -translate-x-1/2 -translate-y-1/2 items-center justify-center">
```

Comment:
Tested it with a badge with icon and it's not messing up its content.

```
() => (
  <OrbitProvider theme={defaultTheme} useId={React.useId}>
    <Stack spacing="1200">
      <Separator label={<FlightDirect />} />
      <Separator
        label={
          <Stack justify="center">
            <FlightDirect />
          </Stack>
        }
      />
      <Separator
        label={
          <Badge icon={<Icons.Airplane />} type="warning">
            foo bar
          </Badge>
        }
      />
    </Stack>
  </OrbitProvider>
);
```

<img width="750" alt="image" src="https://github.com/user-attachments/assets/0af0da9f-6a58-4acb-b50d-a2afbadb33f3" />